{{cookiecutter.project_slug}}/.github/workflows/expo-main.yml

```diff
@@ -37,3 +59,10 @@ jobs:
 
       - name: 🚀 Publish Staging Preview
         run: cd mobile && eas update --branch=staging --non-interactive --auto
+        env:
+          BACKEND_SERVER_URL: ""
+          ROLLBAR_ACCESS_TOKEN: ""
+          SENTRY_PROJECT_NAME: ""
+          SENTRY_DSN: ""
+          SENTRY_AUTH_TOKEN: ""
```

Comment:
Not using secrets?

Reply:
Non of these are 'secrets' they are hard coded values that are added in the code repo. The only one that should be considered secret is the Sentry auth token. For the time being I have added comments in the readme about how to set this up. I'll be doing a clean up on a separate PR!